scm/mock_test.go

```diff
@@ -939,26 +925,82 @@ func TestMockCreateCourse(t *testing.T) {
 		}
 		return false
 	}
-	for _, r := range wantRepos {
-		if !found(r, repos) {
-			t.Errorf("expected repository %s to be found", r)
-		}
-	}
 
-	wantTeams := map[uint64]*scm.Team{
-		1: {
-			ID:           1,
-			Name:         scm.TeachersTeam,
-			Organization: qtest.MockOrg,
+	tests := []struct {
+		name      string
+		opt       *scm.CourseOptions
+		wantRepos int
+		wantTeams map[uint64]*scm.Team
+		wantErr   bool
+	}{
+		{
+			"invalid opt, missing org ID",
```

Comment:
It would be better if these table entries used the field labels like: `name`, `opt` etc.